test/compilable/test18670.d

```diff
@@ -0,0 +1,12 @@
+// REQUIRED_ARGS: -dip1000
+
+// https://issues.dlang.org/show_bug.cgi?id=18670
+
+void foo() {
```

Comment:
Missing `@safe`

Reply:
The ICE happened without @safe and I thought the general policy is to copy/paste the submitted samples to the testsuite?

Reply:
Damn, another case were DIP1000 is not doing what it's supposed to do (nothing DIP1000 related should be checked outside of `@safe` code).